myosuite/envs/myo/myodm/myodm_v0.py

```diff
@@ -1,11 +1,11 @@
 """ =================================================
 Copyright (C) 2018 Vikash Kumar
 Author  :: Vikash Kumar (vikashplus@gmail.com), Sudeep Dasari (sdasari@andrew.cmu.edu), Vittorio Caggiano (caggiano@gmail.com)
-Source  :: https://github.com/MyoHub/myosuite
+Source  :: https://github.com/vikashplus/robohive
```

Comment:
source should be where the file is located .. why does it point to robohive?

Reply:
SG